HM3CoreKill/HM3CoreKill/ck/HM3InGameTools.cpp

```diff
@@ -330,6 +332,55 @@ namespace ck
 					ImGui::Text("Location pointer at 0x%.8X", currentActor->ActorInformation->location);
 					ImGui::Text("Position:"); ImGui::SameLine(0.f, 4.f); ImGui::InputFloat3("", actorPosition); 
 
+					{
+						ImGui::Text("Set animation: "); ImGui::SameLine(0.f, 5.f);
+
+						static std::string item_current = "(None)";            // Here our selection is a single pointer stored outside the object.
+						static ioi::hm3::ZAnimationInfo* current_anim = nullptr;
+
+						if (ImGui::BeginCombo("CXX", item_current.c_str())) // The second parameter is the label previewed before opening the combo.
```

Comment:
Possible bug of ImGui, empty text brokes the ComboBox logic. Need to use here some empty string or use specific flag (need to read docs carefully for this)